lib/manager/gradle/index.js

```diff
@@ -18,9 +18,10 @@ async function extractAllPackageFiles(config, packageFiles) {
   // Gradle needs all files to be written to disk before we parse any as some files may reference others
   // But if we're using gitFs then it's not necessary because they're already there
   if (!config.gitFs) {
-    for (const packageFile of packageFiles) {
-      const localFileName = path.join(config.localDir, packageFile);
-      const content = await platform.getFile(packageFile);
+    const files = [...packageFiles, 'gradlew'];
```

Comment:
why are you saving the gradlew file if no gitFS but then below you have an if to not use it if no gitFs?

Reply:
As stated in #3135, the gradle wrapper (`gradlew`) should be privileged to system binary `gradle`. So I think it makes sense that, even when you want to use docker binaries, if the wrapper is found you use it. I'm not a lot familiar with gradle but I never seen a reason to use the system gradle when the wrapper was available.